### PR TITLE
Switch from contoml to toml

### PR DESCRIPTION
**Pull request by fridex**
Fixes: https://github.com/thoth-station/thamos/issues/232

### CODE REVIEW

**Comment by fridex** [comment]
The generated Pipfile will be formatted in a different way slightly. Let's switch to toml, the upstream contoml seems to be not responsive anyway [1].

[1] https://github.com/jumpscale7/python-consistent-toml/issues/24

File: Pipfile [hunk 1]
```diff
@@ -16,9 +16,9 @@ yaspin = "*"
 texttable = "*"
 pyyaml = "*"
 requests = "*"
-contoml = "*"
 termcolor = "*"
 invectio = "*"
+toml = "*"
```
Comment (goern) [inline Pipfile]: could you create a micro-test that verifies that this works with nested empty dict?! ;)

Comment (fridex) [reply Pipfile]: Done. We should enable zuul's test.

**Comment by fridex** [comment]
recheck

**Pull request merged**